apps/els_lsp/src/els_dt_document.erl

```diff
@@ -201,3 +205,14 @@ wrapping_functions(Document, Line, Column) ->
 wrapping_functions(Document, Range) ->
   #{start := #{character := Character, line := Line}} = Range,
   wrapping_functions(Document, Line, Character).
+
+-spec find_candidates(binary()) -> [uri()].
+find_candidates(Pattern) ->
```

Comment:
Let's say there's room for improvements here...

Reply:
great start, though :)